### PR TITLE
Add Act 1, Scene 1 of Gameroom walkthrough

### PR DESCRIPTION
**Pull request by shannynalayna**

### CODE REVIEW

**Comment by vaporos** [comment]
> Should the gameroom walkthough be in an example/ directory? Possibly under the 0.4/ directory as the goal is to eventually bring the walkthrough up to date with the 0.4 version of splinter?

Probably, but instead of debating it in this PR, I suggest merging it and move it as we sort out the rest of the navigation details on the site.

File: docs/gameroom_walkthrough/act_1/behind_the_scenes/behind_scene_1.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+``` json
+    POST /users/authenticate
+    {
+      "email": "alice@acme.com",
+      "hashedPassword": "8e066d41...d99ada0d"
+    }
+```
+
+The UI does not reveal the user's password to the REST API because the password
+is used to encrypt signing keys.
```
Comment (chenette) [inline docs/gameroom_walkthrough/act_1/behind_the_scenes/behind_scene_1.md:25]: The original doc had cross references (links to other sections of the doc).  I strongly recommend adding them now as plain text.  Later, when the full doc is available, we can activate the links.

File: docs/gameroom_walkthrough/act_1/behind_the_scenes/behind_scene_1.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      hashed_password TEXT NOT NULL
+    );
+```
+
+Alice's public and private key pair was created during registration and was
+added to the Acme Gameroom database. The database has the following entry:
+
+| email | hashed_password | public_key | encrypted_private_key |
+| :---: | :---: | :---: | :---: |
+|alice@acme.com|56ec82cb...480cad32|0384781f...5a7e4998|{\"iv\":...cgXrm\"}|
```
Comment (chenette) [inline docs/gameroom_walkthrough/act_1/behind_the_scenes/behind_scene_1.md:52]: Not a blocker for this PR, but we need to find a way to disable automatic links for this and other example email addresses and URLs in the document.  Users should not be able to click these items (plus, the blue is distracting).

Comment (chenette) [reply docs/gameroom_walkthrough/act_1/behind_the_scenes/behind_scene_1.md:52]: According to this helpful person, the fix is to add <span> tags around the item that should not be a link. I have not tested this yet.

https://eureka.ykyuen.info/2014/07/04/markdown-disable-url-auto-url-link-in-github-flavored-markdown/

Comment (shannynalayna) [reply docs/gameroom_walkthrough/act_1/behind_the_scenes/behind_scene_1.md:52]: Would definitely agree, wasn't sure of a way to take care of this in just markdown.

Comment (chenette) [reply docs/gameroom_walkthrough/act_1/behind_the_scenes/behind_scene_1.md:53]: Also, the text in the table is supposed to be fixed-width font (surrounded by back-ticks).

File: docs/gameroom_walkthrough/act_1/behind_the_scenes/behind_scene_1.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    }
+```
+
+### I-1.5. Acme UI requests a list of invitations
+Next, the Acme Gameroom UI requests the list of gameroom invitations. In this
+scenario, Alice has no invitations, so the list is empty. Later, the walkthrough
+will show what happens when a user has unaccepted invitations.
+
+1. When Alice logs in, the UI makes a call to the Gameroom REST API for the list
+of invitations (also called circuit proposals).
```
[the remark this answers is not in the document]
Comment (shannynalayna) [reply docs/gameroom_walkthrough/act_1/behind_the_scenes/behind_scene_1.md:110]: Will fix in my pr to update this info

**Pull request merged**